File: deadinside.cs
// дединсайдам привет, остальным соболезную
// by https://yuni.su

using System;

namespace ConsoleApp2
{
    class Program
    {
        static void Main(string[] args)
        {
            for (int i = 1000, i > 2; i-=3)
            {
                Console.WriteLine(i + " - 3 = " + (i - 3));
                System.Threading.Thread.Sleep(500);
            }
        }
    }
}

File: deadinside.java
// дединсайдам привет, остальным соболезную
// by https://yuni.su

public class Main {
    public static void main(String[] args) throws InterruptedException {
        int result = 1000;
        while ((result -7) >= 0){
            result -= 7;
            System.out.println(result);
            Thread.sleep(3000);
        }
    }
}